package.json

```diff
@@ -220,8 +220,10 @@
     "@typescript-eslint/eslint-plugin": "^5.39.0",
     "@typescript-eslint/parser": "^5.39.0",
     "babel-jest": "^28.1.0",
+    "base-64": "^1.0.0",
```

Comment:
Don't we want to have the types? 
https://www.npmjs.com/package/@types/base-64

Reply:
that is not necessary for now since it is used in only one line.